requirements/detection_test.txt

```diff
@@ -1,4 +1,4 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
-faster-coco-eval >=1.6.2, <1.7.0
+faster-coco-eval >=1.6.2, <1.7.0; sys_platform != "darwin"  # compilation error, todo: try to remove this OS contratin with next package bump
```

Comment:
Before killing darwin support, can we test the new version again?
https://pypi.org/project/faster-coco-eval/1.6.3/
I posted it, on my local macos m1 it builds without problems.

Reply:
```suggestion
faster-coco-eval >=1.6.3, <1.7.0
```